cmd/tools/extract_actions/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"go/ast"
+	"go/importer"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"log"
+	"os"
+)
+
+func main() {
+	inFile := os.Args[1]
+	outFile := os.Args[2]
+
+	out, err := os.Create(outFile)
```

Comment:
Move the creation after the inFile processing

Reply:
Not done: if I cannot create the file, that's the first thing I should complain about!